index.html

```diff
@@ -0,0 +1,981 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+      integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.7.0/css/all.css"
+      integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>The News Week Clone</title>
+  </head>
+  <body>
+    <header class="page-header fixed-top bg-white">
+      <div class="container-fluid p-0">
+        <nav
+          class="navbar navbar-expand-md navbar-dark bg-danger d-flex flex-nowrap justify-content-between"
```

Comment:
- [ ] The header should be fixed, yes, but you should still be able to see the first articles of the main section properly.
Currently the header is blocking the vision of the top articles, please see how it looks in the original page and fix it by adding a margin/padding to the main section, under the header.

How it currently looks, notice how it blocks the top content of the main section:
![Screenshot_2021-02-04 The News Week Clone](https://user-images.githubusercontent.com/16193212/106883807-3fcb3200-66bf-11eb-9ba0-dc235d1d117d.png)

Reply:
changes updated as requested